README.md

```diff
@@ -224,29 +242,24 @@ cd che
 git checkout openshift-connector
 ```
 
-* Set some environment variables (or run `. ./scripts/setenv-for-build.sh`)
+* Set some environment variables:
+    * `UPSTREAM_CHE_PATH` points to the local directory where the official Che repo was checked out
+    and is *mandatory* for some scripts
 
 ```bash
-export CHE_IMAGE_REPO=rhche/che-server
-export CHE_IMAGE_TAG=nightly
-export GITHUB_REPO="/Users/mariolet/Github/che"
-eval $(minishift docker-env)
+export UPSTREAM_CHE_PATH="/Users/mariolet/Github/che"
 ```
 
-* CHE_IMAGE_REPO and CHE_IMAGE_TAG are the Docker image name and tag which will be built.
-* GITHUB_REPO points to the local directory where the official Che repo was checked out
-
-### Build Che
-
-In the rh-che repository, build Che by executing the following script:
+    * `DOCKER_HUB_NAMESPACE`, `DOCKER_HUB_USER` and `DOCKER_HUB_PASSWORD` can be overridden to point
```

Comment:
This bullet point is not parsed well by the markdown interpreter. It's not at the same level of `UPSTREAM_CHE_PATH` and it looks like `code`